sdk/vmwarecloudsimple/ci.mgmt.yml

```diff
@@ -13,9 +13,7 @@ trigger:
       - release/*
   paths:
     include:
-      - sdk/vmwarecloudsimple/
-    exclude:
-      - sdk/vmwarecloudsimple/microsoft-azure
+      - sdk/vmwarecloudsimple/mgmt-
```

Comment:
Are you sure this path prefix works? I don't believe DevOps allows such syntax so I'd suggest you validate it.

Reply:
The matching logic for pipelines is effectively a starts-with check. So if your folkders start with ```sdk/vmwareclouresimple/mgmt-``` it should match (and in this case exclude) them.

Reply:
... you just can't/don't need the wildcard symbols like ```*``` etc.

Reply:
@weshaggard @mitchdenny 

I've tried part of path in my own repo. It indeed does not work.

I set the "include" to "src/ma", not work, "src/main", works.

I will revert it back to "include" + "exclude" setting. Even if "exclude" part not working correctly, it still be better compared to "include" part not working.